guides/common/attributes.adoc

```diff
@@ -139,6 +141,7 @@ ifeval::["{build}" == "foreman-deb"]
 :package-clean: apt-get clean
 :package-remove: apt-get remove
 :package-install-project: apt-get install
+:package-update-project: apt-get update
```

Comment:
`apt-get update` only updates the metadata. You're looking for `apt-get upgrade`.

Technically speaking `apt-get upgrade` is equivalent to `yum update --cacheonly`

Reply:
This is a good catch, can we fix this @spetrosi please?

Reply:
Please merge #255 to fix it @lzap